crawler/helpers/cherrypicker_test_data.py

```diff
@@ -0,0 +1,157 @@
+import csv
+import logging
+import os
+import random
+from datetime import datetime
+
+import requests
+
+from crawler.types import Config
+
+logger = logging.getLogger(__name__)
+
+
+def make_coords_list():
+    for column in "ABCDEFGH":
+        for row in range(1, 13):
+            yield "%s%02d" % (column, row)
+
+
+WELL_COORDS = list(make_coords_list())
+PLATE_SIZE = len(WELL_COORDS)
+
+SAMPLES_FILE_HEADERS = [
+    "Root Sample ID",
+    "Viral Prep ID",
+    "RNA ID",
+    "RNA-PCR ID",
+    "Result",
+    "Date Tested",
+    "Lab ID",
+    "CH1-Target",
+    "CH1-Result",
+    "CH1-Cq",
+    "CH2-Target",
+    "CH2-Result",
+    "CH2-Cq",
+    "CH3-Target",
+    "CH3-Result",
+    "CH3-Cq",
+    "CH4-Target",
+    "CH4-Result",
+    "CH4-Cq",
+]
+
+BARACODA_PREFIX = "TEST"
+
+
+def flatten(nested_list: list) -> list:
+    return [item for sublist in nested_list for item in sublist]
+
+
+def generate_baracoda_barcodes(config: Config, num_required: int) -> list:
+    baracoda_url = f"{config.BARACODA_BASE_URL}/barcodes_group/{BARACODA_PREFIX}/new?count={num_required}"
+    response = requests.post(baracoda_url, data={})
```

Comment:
Is `data` required?

Reply:
Not sure about that one, I copied from other code I'd found, into my script, and Stuart's copied that into here.